h/groups/views.py

```diff
@@ -15,13 +15,15 @@
 
 @view_config(route_name='group_create',
              request_method='GET',
-             renderer='h:groups/templates/create.html.jinja2',
-             permission="authenticated")
+             renderer='h:groups/templates/create.html.jinja2')
 def create_form(request):
     """Render the form for creating a new group."""
     if not request.feature('groups'):
         raise exc.HTTPNotFound()
 
+    if request.authenticated_userid is None:
+        raise exc.HTTPNotFound()
```

Comment:
> It's a match predicate for the route, so it would have been a 404.

You're wrong about that, @nickstenning. If the route is protected with a permission predicate it causes an unauthorized response. The only reason we show a 404 is because we have a catch-all for unauthorized that turns it into a 404.